content/en/docs/setup/_index.md

```diff
@@ -11,3 +11,90 @@ aliases:
     - /docs/setup/kubernetes/download/
 keywords: [kubernetes,install,quick-start,setup,installation]
 ---
+
```

Comment:
Remove lines 14 to 100. They were removed in a previous cleanup PR which this is reverting.